servers/rendering/rendering_device_commons.h

```diff
@@ -549,6 +580,29 @@ class RenderingDeviceCommons : public Object {
 		SHADER_STAGE_COMPUTE_BIT = (1 << SHADER_STAGE_COMPUTE),
 	};
 
+	enum ShaderLanguage {
+		SHADER_LANGUAGE_GLSL,
+		SHADER_LANGUAGE_HLSL
+	};
+
+	enum ShaderLanguageVersion {
+		SHADER_LANGUAGE_VULKAN_VERSION_1_0 = (1 << 22),
+		SHADER_LANGUAGE_VULKAN_VERSION_1_1 = (1 << 22) | (1 << 12),
+		SHADER_LANGUAGE_VULKAN_VERSION_1_2 = (1 << 22) | (2 << 12),
+		SHADER_LANGUAGE_VULKAN_VERSION_1_3 = (1 << 22) | (3 << 12),
+		SHADER_LANGUAGE_OPENGL_VERSION_4_5_0 = 450
```

Comment:
There's Vulkan 1.4 since recently, does it need to be added here?

Reply:
Sure, adding it.